components/weekoverview_components/activity_components/ActivityAddButton.tsx

```diff
@@ -29,7 +29,7 @@ const styles = StyleSheet.create({
     justifyContent: "center",
     width: 60,
     height: 60,
-    backgroundColor: colors.lightGreen,
+    backgroundColor: colors.gray,
```

Comment:
why has this been changed to gray? It followed the old colour theme, if we want to update the colours used, please update the colour theme

Reply:
Oh.
That one is on me. It appears that I took the colors from the wrong version. 
On it.